app/models/solidus_paypal_commerce_platform/paypal_order.rb

```diff
@@ -103,5 +119,9 @@ def price(amount)
         value: amount
       }
     end
+
+    def greater_than_2_10
+      ::Spree.solidus_gem_version >= Gem::Version.new(2.11)
+    end
```

Comment:
Thoughts on adding a `?` to the method name?
nit: `Gem::Version.new` should probably take a string instead of a float

Reply:
Changed.